tools/cassandra/cqlclient_test.go

```diff
@@ -79,7 +79,7 @@ func (s *CQLClientTestSuite) TestCQLClient() {
 func newTestCQLClient(keyspace string) (*cqlClient, error) {
 	return newCQLClient(&CQLClientConfig{
 		Hosts:       environment.GetCassandraAddress(),
-		Port:        defaultCassandraPort,
+		Port:        environment.GetCassandraPort(),
```

Comment:
not blocking, just thinking out loud -- 

`defaultCassandraPort` is still defined and used in some places
```
~/src/temporal-alex2 (feature/get-cassandra-port|✔) $ grep -r  defaultCassandraPort *
tools/cassandra/handler.go:		config.Port = defaultCassandraPort
tools/cassandra/cqlclient.go:	defaultCassandraPort = 9042
tools/cassandra/main.go:			Value:  defaultCassandraPort,
```

In the meantime, GetCassandraPort defines/uses its own default port value:

https://github.com/temporalio/temporal/blob/b70271b98d49927953c6916a56ceac573002916c/environment/env.go#L159

https://github.com/temporalio/temporal/blob/b70271b98d49927953c6916a56ceac573002916c/environment/env.go#L41-L42

i wonder if it would be beneficial / reasonably easy to consolidate the two values.